lib/capybara/rspec/features.rb

```diff
@@ -21,9 +23,12 @@ def self.included(base)
           alias :background :before
           alias :scenario :it
           alias :xscenario :xit
+          alias :fscenario :fit
           alias :given :let
           alias :given! :let!
           alias :feature :describe
+          alias :xfeature :xdescribe
```

Comment:
The base version of RSpec we currently claim support for (2.2) doesn't support xdescribe -- don't add any of these changes for RSpec 2 - capybara will no longer support it in the next major release anyway